scalalib/src/Dep.scala

```diff
@@ -93,9 +99,9 @@ object Dep {
     (module.split(':') match {
       case Array(a, b, c) => Dep(a, b, c, cross = empty(platformed = false))
```

Comment:
Didn't set `platformed = true` for java deps since I see them as "raw" it means what you write is what you get.
Also there is no such thing as a platformed java dependency.